SConstruct

```diff
@@ -422,14 +438,23 @@ if platform == "darwin":
 		binary = path.join(install_dir, targ + '.app', 'Contents/MacOS', targ)
 		env.Command(Dir(target_dir), binary, [Delete(target_dir), bundle_libraries_for])
 elif platform == "win32":
-	bundled_libs = Split("""
+	bundled_libs += Split("""
 		libsndfile-1
 		openal32
 		sfml-audio-2
 		sfml-graphics-2
 		sfml-system-2
 		sfml-window-2
 		zlib1
+		freetype
+		vorbis
+		vorbisfile
+		vorbisenc
+		ogg
+		FLAC
+		bz2
+		brotlidec
+		brotlicommon
```

Comment:
Something seems to be wrong here. What on earth is "brotli", and why is bz2 being bundled when we don't use it?

I don't think we use vorbis/ogg/FLAC, either, though it wouldn't hurt to include it for use by mods someday. Including freetype is fine.

Reply:
I agree it seems wrong, but the Visual Studio build outputs all of those DLLs, and the scons build won't launch without them bundled.

Reply:
What do you mean it "outputs" all of those DLLs?

Reply:
They're all in the build directory Visual Studio outputs, and if you rename one of them to something else, the binary won't launch.

Reply:
So then the question is, what exactly is causing them to be required? They shouldn't be required, since we don't use them anywhere. Are they transitive dependencies of something else? If so, the something else should be built in a way that doesn't require them.